### DISCUSSION TITLE
hangman game does not appear

### DISCUSSION THREAD

**Discussion by Amanda1e** [Monika-After-Story/MonikaModDev#10042] [general]
I already unlocked the game, and Monika's dialogue saying "I hope this is not in bad taste" etc etc
however, in one part the game "hangs" (that gray screen appears with the folders, lines and the rest of what caused the problem) right when monika was going to say "so I added the hangman game"
I already restarted the game but it still doesn't appear
and I already gave the NOU if that makes any difference
Can someone help me??

**Comment by Hannah-Realgames** [comment]
Hey! So I had the same Problem! 
For me it fixed it after clicking/opening it like 9 Times because I wondered why its not opening and I waited. 

Then it works. Maybe just give it some trys and try to open it and wait a bit.
